sidebars.js

```diff
@@ -152,6 +153,7 @@ module.exports = {
                 "competitive-programming/anagram-numbers",
                 "competitive-programming/factorial-of-large-number",
                 "competitive-programming/max-number-from-array",
+                "competitive-programming/first-occurrence-1"
```

Comment:
Missing comma here. Fix this and the build will pass

Reply:
Done!! Checks passing!!